JourneysSquad/JourneysSquad/Model/BillStorage/LinkedListNode.swift

```diff
@@ -0,0 +1,17 @@
+/// Represents a node in a linked list.
+final class LinkedListNode<T> {
```

Comment:
```suggestion
final class DoubleLinkedListNode<T> {
```

according to implementation?

Reply:
Fixed.